src/main/scala/com/danielasfregola/twitter4s/http/serializers/CustomFormats.scala

```diff
@@ -65,8 +65,9 @@ private[twitter4s] case object DisconnectionCodeSerializer
 private[twitter4s] case object ProfileImageSerializer
     extends CustomSerializer[ProfileImage](format =>
       ({
-        case JString(n) => ProfileImage(n)
-        case JNull      => null
+        case JString("") => null
```

Comment:
could you add a comment here to remind why we do it?

Reply:
comment added